contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,202 @@
+---
+date: 2020-08-18
+title: Why You May Not Need a Sales Team
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+<!-- ![PLG Banner](../images/plg-header.png) -->
+
+Let's say you are the founder of a new tech startup. 
+
+A few months back, you and a couple of friends (all developers) built an amazing tool and have now received Seed funding. 
+
+As a result, you're under some pressure to grow. You took a few million from some VCs here and there, and they want to see some results.
+
+Since the realization of needing to grow hit you, you've started to lose some sleep. 
+
+Before, you were just a group of friends working your ass off to build an amazing product, with some revenue coming in from your immediate network. Now you actually have to focus on ramping up your sales.
+
+You call in a founders meeting to discuss next steps, and a conclusion is reached: you can't spend all your budget on engineers, which is what you wanted to do. 
+
+Instead, you need to hire some Sales people. Or some Marketing people. Or both. Whoever can get you some users that pay for your product. Or do you?
+
+## Getting People to Pay You
+
+VC funding or not, all startups will eventually need to get paid for what they do. 
+
+Irrespective of how cool your idea is, or how helpful it may be, if you're not getting money in, you can't push it forward. 
+You need customers.
+
+Traditionally, there are two ways to do this: Sales and Marketing.
+
+Let's take a quick look at how they work.
+
+### Sales-led Growth 🤝
+Sales involves actively going after potential customers and getting them on board with your product or service. This can be done in a variety of ways, from cold calls to leveraging leads. 
+
+Essentially, the assumption is that there are people out there who are willing to pay you for what you do, so you need to do your best to find them and get them to pay you.
+
+### Marketing-led Growth 📈
+While often paired up with Sales, and sometimes even used interchangeably, Marketing is, in some ways, the opposite of Sales. 
```

Comment:
It struck me as weird that "Sales" and "Marketing" are capitalised throughout. They're not proper nouns, so is the intention that you're highlighting them?

Reply:
Yeah this isn't necessary and I could change it. I capitalize them because I view them as teams inside a company. I feel like the meaning of the noun can get tangled with the name of the team otherwise.

Like:

"We should run this decision by Product"
vs.
"We should run this decision by product"

Or:

"I think Engineering can sort that"
vs. 
"I think engineering can sort that

Reply:
Very valid point, and not something I had considered. Although, I felt like the post was talking conceptually so the capitalisation felt weird. E.g. the growth of your company is fuelled by the sales/marketing/product as opposed to the Sales Team etc.

Reply:
Right - I wonder if some could be de-capitalized but then I also wonder if that would be picked up as an inconsistency